packages/api/src/router/widgets/dns-hole.ts

```diff
@@ -22,9 +22,9 @@ export const dnsHoleRouter = createTRPCRouter({
               case "piHole":
                 client = new PiHoleIntegration(integration);
                 break;
-              // case 'adGuardHome':
-              //   client = new AdGuardHomeIntegration(integration);
-              //   break;
+              case "adGuardHome":
+                client = new AdGuardHomeIntegration(integration);
+                break;
               default:
```

Comment:
default option can be removed as we restrict only piHole and adGuardHome in the middleware (`unstable_concat`)
Please update for all 3 occurences